matcher/service/openreview_interface.py

```diff
@@ -977,9 +978,18 @@ def run(self):
                         client_v2, notes[0].id
                     )
                 else:
-                    raise openreview.OpenReviewException(
-                        "Venue request not found"
+                    notes = client_v2.get_notes(
```

Comment:
one way to do this to avoid the hardcoding of the invitation id is to get the domain group and if the domain group has the property `venue_id` then instantiate the venue object.